### PR TITLE
Better maintenance page

### PR DESCRIPTION
**Pull request by Tirokk**
> [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-09T01:55:39Z" title="Monday, September 9th 2019, 3:55:39 am +02:00">Sep 9, 2019</time>_
_Merged <time datetime="2019-09-11T16:25:21Z" title="Wednesday, September 11th 2019, 6:25:21 pm +02:00">Sep 11, 2019</time>_
---

## 🍰 Pullrequest

The idea is to statically generate a single maintenance page with a dedicated `nuxt.config.maintenance.js`. We would still have the `i18n` plugin and the `Styleguide` so the page looks similar to the rest of the application.

### Issues

- None

### Todo

- [X] None

2019-09-10-150649_1920x1080_scrot

### CODE REVIEW

File: webapp/maintenance/source/pages/index.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<template>
+  <transition name="fade" appear>
+    <ds-container width="medium">
```
Comment (Tirokk) [inline webapp/maintenance/source/pages/index.vue:3]: > [alina-beck](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-09-11T13:52:45Z" title="Wednesday, September 11th 2019, 3:52:45 pm +02:00">Sep 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,52 @@
+<template>
+  <transition name="fade" appear>
+    <ds-container width="small">
```


@mattwr18 didn't you say you changed this to `medium`?

Comment (Tirokk) [reply webapp/maintenance/source/pages/index.vue:3]: > [mattwr18](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-11T14:08:53Z" title="Wednesday, September 11th 2019, 4:08:53 pm +02:00">Sep 11, 2019</time>_

---


yes, I did!!

**Review by Mogge — APPROVED** [review]
I don't understand much of what's going on in this PR – but the frontend part looks good and the page is veeery pretty. :smile:

Great work @roschaefer!

**Review by Mogge — APPROVED** [review]
great! love the new maintenance page!!

**Pull request closed without merging**